src/component/Header.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useTodos } from '../utils/TodoContext';
+import classNames from 'classnames';
+import { Errors } from '../types/ErrorsTodo';
+import { USER_ID } from '../api/todos';
+
+export const Header: React.FC = () => {
+  const { todos, addTodo, loading, setLoading, setErrorMessage } = useTodos();
+  const inputRef = useRef<HTMLInputElement | null>(null);
+  const [inputTodo, setInputTodo] = useState('');
+  const [submitting, setSubmitting] = useState(false);
```

Comment:
looks like it's redundant and you can use the isLoading state

Reply:
I remove the submitting state. 
in code test passed, but after the deployed test 'should remove todos with success responses and keep todos with errors' did not pass.
Could you suggest where is the problem?

Reply:
<img width="1327" alt="Screenshot 2024-04-17 at 13 23 38" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/117312348/8962cfe9-c1c3-4f31-8d65-a6d577854dee">

Reply:
now tests pass, but I remove setTodos(todos) and [todos].
Won't this cause errors later on?